ballerina/tests/init.bal

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/time;
+import ballerina/lang.regexp;
+
+configurable string accessKeyId = ?;
+configurable string secretAccessKey = ?;
+configurable string region = ?;
+
+ConnectionConfig config = {
+    accessKeyId: accessKeyId,
+    secretAccessKey: secretAccessKey,
+    region: region
+};
+
+Client amazonSNSClient = check new(config);
+
+string testRunId = regexp:replaceAll(re `[:.]`, time:utcToString(time:utcNow()), "");
+
+configurable string testHttp = ?;
+configurable string testHttps = ?;
+configurable string testEmail = ?;
+configurable string testPhoneNumber = ?;
+configurable string testApplication = ?;
+configurable string testIamRole = ?;
+configurable string testAwsAccountId = ?;
+configurable string testEndpoint = ?;
+ 
+function isArn(string arn) returns boolean {
+    return arn.startsWith("arn:aws:");
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in [020b0a4](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/020b0a40d31509c3e7b499cf2ea5c620bbfc9b2b)